utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java

```diff
@@ -26,7 +27,6 @@
  * @author r.rajasekaran
  * @since 232
  */
-@SuppressWarnings("rawtypes")
 public abstract class MobileDriverUtils {
```

Comment:
since getSize() and getLocation() are for JWP only, so, should we update to use getRect().getDimension() and getRect().getPoint() accordingly to make it work for both JWP and W3C? @jimevanssfdc

Reply:
Probably so. Let me see about that.

Reply:
I've updated this.